crm_request/models/mail_mail.py

```diff
@@ -0,0 +1,25 @@
+# coding: utf-8
+from odoo import models, api
+
+
+class MailMail(models.Model):
+    _inherit = "mail.mail"
+
+    '''
+        Inherit mail.mail to deactivate a linked partner after a send.
+    '''
+
+    @api.multi
+    def _postprocess_sent_message(self, mail_sent=True):
+
+        s = super(MailMail, self)._postprocess_sent_message(mail_sent)
```

Comment:
The method `_postprocess_sent_message` can delete some e-mails. Therefore, the code below will fail (see the error in Travis). I think you should do your method before and call the super at the end.